docs/source/about/guides/linux/headless_ssh.rst

```diff
@@ -0,0 +1,511 @@
+#. `SSH Client Script (Optional)`_
+
+SSH Key Authentication Setup
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Setup your SSH keys with ``ssh-keygen`` and use ``ssh-copy-id`` to authorize remote login to your host.
+   Run ``ssh <user>@<ip_address>`` to login to your host.
+   SSH keys automate login so you don't need to input your password!
+#. Optionally setup a ``~/.ssh/config`` file to simplify the ``ssh`` command
+
+   .. code-block:: text
+
+      Host <some_alias>
+          Hostname <ip_address>
+          User <username>
+          IdentityFile ~/.ssh/<your_private_key>
+
+   Now you can use ``ssh <some_alias>``.
+   ``ssh <some_alias> <commands/script>`` will execute the command or script on the remote host.
+
+Checkpoint
+^^^^^^^^^^
+
+As a sanity check, let's make sure your setup is working so far!
+
+**Test Steps**
+
+With your monitor still plugged into your Sunshine host PC:
+
+#. ``ssh <alias>``
+#. ``~/scripts/sunshine.sh``
+#. ``nvidia-smi``
+
+   You should see the sunshine and Xorg processing running:
+
+   .. code-block:: console
+
+       $ nvidia-smi
+       Tue Aug 29 18:38:46 2023
+       +---------------------------------------------------------------------------------------+
+       | NVIDIA-SMI 535.104.05             Driver Version: 535.104.05   CUDA Version: 12.2     |
+       |-----------------------------------------+----------------------+----------------------+
+       | GPU  Name                 Persistence-M | Bus-Id        Disp.A | Volatile Uncorr. ECC |
+       | Fan  Temp   Perf          Pwr:Usage/Cap |         Memory-Usage | GPU-Util  Compute M. |
+       |                                         |                      |               MIG M. |
+       |=========================================+======================+======================|
+       |   0  NVIDIA GeForce RTX 3070        Off | 00000000:01:00.0  On |                  N/A |
+       | 30%   46C    P2              45W / 220W |    549MiB /  8192MiB |      2%      Default |
+       |                                         |                      |                  N/A |
+       +-----------------------------------------+----------------------+----------------------+
+
+       +---------------------------------------------------------------------------------------+
+       | Processes:                                                                            |
+       |  GPU   GI   CI        PID   Type   Process name                            GPU Memory |
+       |        ID   ID                                                             Usage      |
+       |=======================================================================================|
+       |    0   N/A  N/A      1393      G   /usr/lib/Xorg                                86MiB |
+       |    0   N/A  N/A      1440    C+G   sunshine                                    293MiB |
+       +---------------------------------------------------------------------------------------+
```

Comment:
```suggestion
   .. code-block:: bash

      nvidia-smi

   *Output:*

   .. code-block:: console

       +---------------------------------------------------------------------------------------+
       | NVIDIA-SMI 535.104.05             Driver Version: 535.104.05   CUDA Version: 12.2     |
       |-----------------------------------------+----------------------+----------------------+
       | GPU  Name                 Persistence-M | Bus-Id        Disp.A | Volatile Uncorr. ECC |
       | Fan  Temp   Perf          Pwr:Usage/Cap |         Memory-Usage | GPU-Util  Compute M. |
       |                                         |                      |               MIG M. |
       |=========================================+======================+======================|
       |   0  NVIDIA GeForce RTX 3070        Off | 00000000:01:00.0  On |                  N/A |
       | 30%   46C    P2              45W / 220W |    549MiB /  8192MiB |      2%      Default |
       |                                         |                      |                  N/A |
       +-----------------------------------------+----------------------+----------------------+

       +---------------------------------------------------------------------------------------+
       | Processes:                                                                            |
       |  GPU   GI   CI        PID   Type   Process name                            GPU Memory |
       |        ID   ID                                                             Usage      |
       |=======================================================================================|
       |    0   N/A  N/A      1393      G   /usr/lib/Xorg                                86MiB |
       |    0   N/A  N/A      1440    C+G   sunshine                                    293MiB |
       +---------------------------------------------------------------------------------------+
```

This might look better, and be easier for user to copy and paste the command.

![image](https://github.com/LizardByte/Sunshine/assets/42013603/c89c4be1-5b13-4503-a37a-efeec010a51d)